src/components/form/PartnerDataTextField.js

```diff
@@ -0,0 +1,73 @@
+import React, {useState} from 'react';
+import {makeStyles} from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+
+const useStyles = makeStyles(() => ({
+  textField: {
+    backgroundColor: '#ffffff',
+    width: '100%',
+  },
+
+  cssLabel: {
+    color: '#1d3557',
+    '&$cssFocused': {
+      color: '#1d3557',
+    },
+  },
+
+  cssOutlinedInput: {
+    '&$cssFocused $notchedOutline': {
+      color: '#1d3557',
+      textColor: `#1d3557`,
+      borderColor: `#1d3557`,
+    },
+    color: '#1d3557',
+  },
+
+  cssFocused: {
+    color: '#1d3557',
+  },
+
+  notchedOutline: {
+    borderWidth: '1px',
+    borderColor: '#1d3557',
+    textColor: '#1d3557',
+  },
+}));
+
+export default function GooglerDataTextField(props) {
+  const classes = useStyles();
+  const [data, setData] = useState('');
+
+  // function: Event => ()
```

Comment:
I thought (for maintaining and reading purpose) we could use such syntax for comments regarding types.
I encountered few bugs mainly because of strange type and maybe such thing would make them appear less often.

Reply:
@xmxwx @semiluna what do you think about it?

Reply:
Flow seems like a good quick fix for this, I want to use it as well.

Reply:
If this choice was entirely mine to make, I think I'd use this occasion to learn TypeScript. But I'd be completely fine with you choosing Flow or ESLint+JSDoc.

BTW. If you were to pick Flow, would you prefer to use it in the [JS-compatible (compile-less, comment-only) way](https://flow.org/en/docs/types/comments/)?

Reply:
Flow it is.